ciao-cli/image.go

```diff
@@ -65,6 +71,8 @@ func (cmd *imageAddCommand) parseArgs(args []string) []string {
 	cmd.Flag.StringVar(&cmd.id, "id", "", "Image UUID")
 	cmd.Flag.StringVar(&cmd.file, "file", "", "Image file to upload")
 	cmd.Flag.StringVar(&cmd.template, "f", "", "Template used to format output")
+	cmd.Flag.BoolVar(&cmd.public, "public", false, "Public Image")
```

Comment:
I see, internal is overloaded.  It's used to indicate an internal image and for tags.  Is there any reason why we can't have tags on a public or private image?